packages/embla-carousel-pattern/src/CarouselDotButton.css

```
@@ -0,0 +1,46 @@
+.saltCarouselDotButton {
+  position: relative; /* Position relative to contain the :after element */
+  width: var(--salt-size-selectable);
+  height: var(--salt-size-selectable);
+  border: none; /* Remove border, use box-shadow for ring effect */
+  box-shadow: inset 0 0 0 1px var(--salt-selectable-borderColor); /* Inset ring effect */
+  -webkit-tap-highlight-color: var(--salt-selectable-borderColor-hover);
+  -webkit-appearance: none;
+  appearance: none;
+  background-color: transparent;
+  touch-action: manipulation;
+  display: inline-flex;
+  text-decoration: none;
+  cursor: pointer;
+  padding: 0;
+  margin: 0;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
```

This should probably be:

```suggestion
  border-radius: var(--salt-palette-corner-strongest);
```